packages/cli/README.md

```diff
@@ -127,3 +127,82 @@ jobs:
       - run: npm ci
       - run: npx code-pushup autorun --upload.apiKey=${{ secrets.PORTAL_API_KEY }}
 ```
+
+## CLI commands and options
+
+### Global Options
+
+| Option           | Type      | Default                 | Description                                                            |
+| ---------------- | --------- | ----------------------- | ---------------------------------------------------------------------- |
+| **`--progress`** | `boolean` | `true`                  | Show progress bar in stdout.                                           |
+| **`--verbose`**  | `boolean` | `false`                 | When true creates more verbose output. This is helpful when debugging. |
+| **`--config`**   | `string`  | `code-pushup.config.js` | Path to the config file, e.g. code-pushup.config.js                    |
+
+### Common Command Options
+
+| Option                      | Type                 | Default  | Description                                          |
+| --------------------------- | -------------------- | -------- | ---------------------------------------------------- |
+| **`--persist.outputDir`**   | `string`             | n/a      | Directory for the produced reports.                  |
+| **`--persist.filename`**    | `string`             | `report` | Filename for the produced reports without extension. |
+| **`--persist.format`**      | `('json' \| 'md')[]` | `json`   | Format(s) of the report file.                        |
+| **`--upload.organization`** | `string`             | n/a      | Organization slug from portal.                       |
+| **`--upload.project`**      | `string`             | n/a      | Project slug from portal.                            |
+| **`--upload.server`**       | `string`             | n/a      | URL to your portal server.                           |
+| **`--upload.apiKey`**       | `string`             | n/a      | API key for the portal server.                       |
+
+> [!NOTE]  
+> All common options can be specified in the configuration file as well. CLI arguments take precedence over configuration file options.
```

Comment:
Perfect :ok_hand: Also, I don't know GitHub's Markdown supports this `[!NOTE]` syntax, very nice :+1: